test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/MiddlewareAnalysisSampleTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
My wording was wonky there, I meant do we actually need the functional test I just wrote because it's really only testing that we haven't broken super basic routing, not do we still need the sample itself.